source/exploring_catalog_and_datasets/01_looking_up_the_catalog/catalog.rst

```diff
@@ -1,79 +1,179 @@
 
-Navigating in the catalog
--------------------------
+The structure of a dataset catalog however remains the same no matter the portal.
 
-The main part of the catalog shows a series of cards representing each dataset available on the platform. On each card, the dataset's metadata are displayed, along with quick links to visualize the data.
+.. image:: images/catalog_fullinterface.png
+   :alt: Full interface of the catalog, with both sides
 
-.. image:: images/explore_card.png
-   :alt: Dataset card
+On the left side (1), are displayed:
 
-On this example, the card is split in two parts. On the left part, there is a theme pictogram, the title, a description, the language and license, keywords and the original publisher. On the right part, there are links to access the data table, a map and analysis tools, and to export or access the API of the dataset.
+- the number of datasets currently displayed,
+- a catalog sort option to change the order of the datasets,
+- a variety of filters for the catalog to find specific datasets,
+- different possibilities to download the catalog.
 
-Like most of the catalog's elements, this card is fully customizable.
+On the right side (2), are displayed the datasets. Each dataset is represented as a rectangle called a "catalog card". On each card, the dataset's main metadata are displayed, along with quick links to visualize the data.
+
+Sorting the catalog
+-------------------
+
+By default, a dataset catalog is sorted by last modification date: in other words, the most recently modified datasets are displayed in first position, at the top of the catalog.
+
+It is possible to change in which order the catalog cards are displayed with the catalog sorting option below the number of datasets, on the left side of the interface.
+
+.. image:: images/catalog_sort.png
+   :alt: Catalog sorting option
+
+The catalog can be sorted by 10 different methods:
+
+- *Alphabetically* (one of the 3 most popular methods "A-Z"): the datasets are sorted by their titles, in an alphabetical order.
+- *Unalphabetically*: the datasets are sorted by their titles, in an unalphabetical order.
+- *Recently modified first* (one of the 3 most popular methods "Modified"): the datasets are sorted from the most recently modified to the least recently modified.
+- *Least mostly modified first*: the datasets are sorted from the least recently modified to the most recently modified.
+- *Most records first*: the datasets are sorted from the one with the biggest amount of records to the one with the least records.
+- *Least records first*: the datasets are sorted from the one with the least records to the one with the biggest amount of records.
```

Comment:
I think we should say "fewest" here, since "records" is countable. However I'm afraid the issue comes from the UI and not the doc.